src/images/tally.svg

```diff
@@ -0,0 +1,31 @@
DygpjnZVxuDm2dpZ5+mxG2PFcWJjMk02v/UGWZwuszBVEuQWXmfPhLzSEHDyH6m6CTSTbEfJfJUPWKzwtjfbiaVxrPrQGlW86btqFLDeaugF5Hf9yXDv8+c215rEXT/s97pVkvwXWyJDtQnkWVpg/FXqjtjIK83/2eyQjYwTW5oyqekZh4f8a3f6a4Ngu8+/kWZq2MEyN7UNx2vpiVZdXWHjN78CxUs8wERdPgpSTZDHNBrjWAk5mPFrphQTHhutnMsycy2Jl1saz6XNZdv9NK32HktWVmxnHtRbQYpuuk3Qg5Og3WfUDHTrCShsETzuUvXQueimGhFza5uprKxuIKOHam7Xt0QfIRXYg64COfB3seWvIWblSDznVBXZefcI24spncD0RIBfZ7ashE9glSdTh9aOTnaDZsTxbYOqtDDcn1K7zGTGNoTsrOq2+WS7+Q7eMjAET+eTiWbt6HkwOA5dPpskuqCmjRlyj92CSvluSmEmVo2T+cPK+cpEd5+k1CVb2QEw8mxCmflEPqBARb9HpG0nRvT+BEatteFqTy02/R7JVwTYT+QZ63POlhebwGOQUXGYvVLazaMQ1dt/RStdwAq22jlCJXOKSrG0m8rt4ngtv186E24oqsDCZZ3Y0S6GCCRpEjwjVNqUmF5kCk9CR37Xw7r9R3qPwQeamrUREcpvKdZKQ4C8X2ThomIi7A0G1dT0L0oxICcnMF1guGvtUoOnQNZxgx+EUuSWbiy+no6mo/1UC2b/GNoEc0BKQyMJjvNOMVCTzRnrKYupMhsVrVkX5DnyiY9W0kVuK7k+jGSm/R7IG5EzEDZwgMvJ4kWG21Z2M/LLD6G8Xq8prxEOemRURcKtWpuws6chdNXjaU7TY9vBbUsXN20SbweARlRP50SBALrJNlAUd+cVzz7+/ntgZWgNsy2XqrQyXT1bW07r2JDh8bDu9B6sz7NUDetz3ttOyJOuMCVyVJPIcJ/Rk7d6LrIzD9Nkss6NZbKvyXhFv0dHC0dVCg57ylYtsvJ00EU5MguA5jOktwzVVemEyz+WTSzgFBRI06ByMY1tuxRN51NBbfH3Z3JRkHTORO0Wc9frRaFM5bO2P8VeXpURoGnTvT7LjUJJEu8H4/ywjX5Q0FgFymZVkvaAjuk+QF5ksHl1Mi21DT1XvAdVRGJZiLTpDd7aSaI9QaasBemrQa2MJxN3xoAn8GsUJHISzkiB4Oko0t3246or33755Na2bTTYBVIgAecgcTZ6DtT3wswiPBkGF7dv08s6Pkr9enUef3oNJWrpMbo7lcWyXtj4TM6Ez+rvq9IpmgNl51O+RbMW6gYyqeobRcStarLNqc3prj0lrz9o5uqUZdXNHs0GLdWK0+15vVOoZpWFqiuDZfgnwOPuuEet9N3lKrR6x3vvxscfPEexnZAHhxHLDWVtZ7/DsarG+T8r8bCijXGfQyn8ob6sWkFf3L0czQp6bNEPIwRuyIWrVJev65r0tyeSpj+iJncS67wmlTS1dxqpxT9M1eg5u3HPoGk6sbovGUjrbh9ZMZn0jKUGIBtsG4iTXrcBae0xausRwqJta6Bp7rPueIIuETI9blft6Ry79wHckGe/C4zSJkxlj+Y1v8N68FqDRetv3/PxVjQGyi/DfpKharO8ZpSgsQbjk9aOeGsLsvrvRUmkIzO67gxyHXZJkv8o6Ha98FJYNVdfwWRkkh76CZjSPwS4KaEaS5NCX/R7PIHfJukHe5WSouJM+41mxeDfxgUq9qm5txAceCvLcdkahiA0+qMrJKPlzDcI1fOws8Z3HMFr38V6A3rqX+M5P+z2eRd4rXqdMjl4L418p1MWbMM0gue+baHp9Tl40CzQ9QWrfPwct6WV/aPCQsxcZ55HPHdP4eKXUU7tJ7P16A0VVfyT2fj1o3+I8ct3ibTzk57dR/JxCnc4hHPBuQqz7XuK7VHzBbD3Edz1IrPtev8cF5Moz+MjXj4yziNA2QVhCeEL2RGLwi8R2fCpaSdUZsb5PkRj8YlCSV5G7WP0LPoFQgo5QPKVQvyuIOHce0Eh+4KuYXXdFJKr6wuy6i+TwVwk4i3YaNU+evnINImMGEWNIhjfxUxY1ndT+bwXZbbYEYn2fJLX/WwQczJ0sykGGEwRYcGWHi44j18pBdD3v/VBNJ7nna8QHao380xjEB75Acs/XgoiwkA/pIOR4PCiBjAwHEX1LhmXEeOm7QZ3Y/QipA4+jGS0KxTUemtFC6sDjJHb/fVAyq9hu2TEcEHJ0ghKoenz+NMJRrgzDvGs8Pv+LytmwUD0+V+IL/THgqEK6YZR8of+a/OSzTeUWSfhCf0ho1k3sCx3qEiVgluzYT5okSsBdwti5RaIEgHBy+IRi2vfjZ9QxfkYJfwc8UkH6KiLL/DdOrtLYU+rQE32YPfdVGlnmDdQssSXUPbJMCZHFXCosvoGzfJlaYy7prXswO27D7Dz6roq5RPrUsYijkaXLopFN4lrz4v6ck8O1V8SSWU+IiGSxTvTUQFk0svZKXwuVRyMDQUJ00cjej9Pni5ri9NVEBkD70RdqiWD5QeROAaLEDGJe2DoRLD2wBxEubriKvO/Hdi3i/ajH78Kox+sRRjzwFBvjgXfUVhywFqS9FA+8Vq8CTR8PvIQRxDxS+10zgQSit3QgtPqSM60YQrmMIUwUdvG7dKckXfyeRVyxDgMTiPnhbK0FlaNeZIBYyD+KWqigrYITCOtrXYRWTzJK6EWEtlEJEtWs+CvwU+RH+2tCFGSUMIKIqCLzLN1MeBt4HvlpmVAQJRklHAQ+gzxORyPxGvAfVGDkCwONIKOE7cDnECaS6m9qhodxhOnjl8ivCdcFjSRjPfoQ2vitiOGsM4J3ziNWRKcRWnf9zMSKaBYyytGH6DGljyyuhwomEf/80qfhwi9Hs5JRjnaE/91S8I+diN6TQOgbJQe+WYQ+MY9Q7kp3TkYROkdT4/8BzWoMvDlxTJUAAAAASUVORK5CYII="/>
```

Comment:
This SVG in particular has a base64 embedded image, which seems odd. It might just be some weird output from Illustrator, but yeah once you figure out a good way to go about optimizing your svgs, I'd keep an eye on this one.